From: Didier Raboud <odyx@debian.org>
Date: Thu, 3 Dec 2020 18:33:35 +0100
Subject: CUPS_EXE_FILE_PERM: also match host_os_name on *-gnu,
 to also catch kfreebsd-gnu

Debian's kfreebsd-gnu architectures (kfreebsd-amd64 and kfreebsd-i386) work as Debian/GNU systems, hence with a 755 CUP_EXE_FILE_PERM, not 555

Bug-Upstream: https://github.com/OpenPrinting/cups/pull/61
---
 config-scripts/cups-defaults.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config-scripts/cups-defaults.m4 b/config-scripts/cups-defaults.m4
index 7a2d35a..9e05bd4 100644
--- a/config-scripts/cups-defaults.m4
+++ b/config-scripts/cups-defaults.m4
@@ -53,7 +53,7 @@ dnl Default executable file permissions
 AC_ARG_WITH(exe_file_perm, [  --with-exe-file-perm    set default executable permissions value, default=0555],
 	CUPS_EXE_FILE_PERM="$withval",
 	[case "$host_os_name" in
-		linux* | gnu*)
+		linux* | gnu* | *-gnu)
 			CUPS_EXE_FILE_PERM="755"
 			;;
 		*)
